### PR TITLE
L0 test for PowerShell on Target Machine Task

### PR DESCRIPTION
**Pull request by vincentdass**

### CODE REVIEW

File: Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1 [hunk 1]
```diff
@@ -26,7 +26,7 @@ Write-Verbose "initializationScriptPath = $initializationScriptPath"
 Write-Verbose "runPowershellInParallel = $runPowershellInParallel"
 Write-Verbose "sessionVariables = $sessionVariables"
 
-. ./PowerShellJob.ps1
+. $PSScriptRoot/PowerShellJob.ps1
 
 import-module "Microsoft.TeamFoundation.DistributedTask.Task.Internal"
```
Comment (mvvsubbu) [inline Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1]: Refer the following tests to understand how to write function based tests

https://github.com/Microsoft/vsts-rm-extensions/blob/master/Extensions/IISWebAppDeploy/Tests/Tasks/IISWebAppMgmt/ManageIISWebApp.Tests.ps1

File: Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1 [hunk 2]
```diff
@@ -61,195 +61,11 @@ if ($enableDetailedLoggingString -ne "true")
     $enableDetailedLoggingString = "false"
 }
 
+. $PSScriptRoot/Utility.ps1
```
Comment (chshrikh) [inline Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1]: better to keep utility import and powerShellJob.ps1 import together at top

**Comment by chshrikh** [comment]
Changes looks fine apart from few comments. Ensure to test the changes on legacy agent once before merging

File: Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1 (hunk 2 above)
Comment (vincentdass) [inline Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1]: done

**Comment by vincentdass** [comment]
Verified with both Coreclr agent and legacy agent for standard and Azure VMs.

**Pull request merged**